packages/atlas/src/components/BenefitCard/BenefitCard.styles.ts

```diff
@@ -0,0 +1,174 @@
+import styled from '@emotion/styled'
+
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { JoyTokenIcon } from '@/components/_icons/JoyTokenIcon'
+import smPattern1 from '@/components/_illustrations/svgs/benefit-card-pattern-sm-1.svg'
+import smPattern2 from '@/components/_illustrations/svgs/benefit-card-pattern-sm-2.svg'
+import smPattern3 from '@/components/_illustrations/svgs/benefit-card-pattern-sm-3.svg'
+import xxsPattern1 from '@/components/_illustrations/svgs/benefit-card-pattern-xxs-1.svg'
+import xxsPattern2 from '@/components/_illustrations/svgs/benefit-card-pattern-xxs-2.svg'
+import xxsPattern3 from '@/components/_illustrations/svgs/benefit-card-pattern-xxs-3.svg'
+import { cVar, media, sizes, square } from '@/styles'
+
+export type Variant = 'compact' | 'full'
+
+export const Pattern = styled.div`
+  position: relative;
+  height: 72px;
+  background-repeat: repeat-x;
+
+  ::after {
+    content: '';
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 108px;
+    height: 100%;
+    background: linear-gradient(90deg, rgb(15 17 20 / 0) 0%, ${cVar('colorCoreNeutral900')} 100%);
+  }
+
+  ${media.sm} {
+    width: 108px;
+    height: unset;
+    background-repeat: repeat-y;
+    align-self: stretch;
+
+    ::after {
+      bottom: 0;
+      left: 0;
+      top: unset;
+      right: unset;
+      width: 100%;
+      height: 56px;
+      background: linear-gradient(180deg, rgb(15 17 20 / 0) 0%, ${cVar('colorCoreNeutral900')} 100%);
+    }
+  }
+`
+
+export const Wrapper = styled.div<{ variant: Variant }>`
+  background-color: ${({ variant }) => cVar(variant === 'full' ? 'colorBackgroundMuted' : 'colorBackground')};
+
+  ${media.sm} {
+    display: flex;
+  }
+
+  &:nth-of-type(3n - 2) {
```

Comment:
I don't think I understand this code and where we actually apply these styles 😅  .  Can we write this clearer? If not could you provide some comments to explain what's going on?